README.md

```diff
@@ -1,2 +1,11 @@
-# relativepath.tech
-The RelativePath Website
+# React + TypeScript + Vite
+
+Template from [here](https://github.com/tailwindtoolbox/Landing-Page/blob/master/index.html).
+
+## Usage
+
+**Assumes you have Node installed. You can get it for your machine [here](https://nodejs.org/en/download)**
+
+- Clone this repo.
+- Navigate to repo and run `npm install` to install dependencies.
```

Comment:
This should be in the setup or deploy automation script.

Reply:
This is contained within the deploy automation script.